big_tests/tests/rest_client_SUITE.erl

```diff
@@ -119,7 +119,7 @@ init_per_suite(Config) ->
     Config1 = dynamic_modules:save_modules(HostType, Config),
     Config2 = rest_helper:maybe_enable_mam(mam_helper:backend(), HostType, Config1),
     MucPattern = distributed_helper:subhost_pattern(muc_light_helper:muc_host_pattern()),
-    dynamic_modules:start(HostType, mod_muc_light,
+    dynamic_modules:restart(HostType, mod_muc_light,
```

Comment:
Could we use `ensure_modules` instead not to rely on the previous state?

Reply:
restart does not rely. It will stop, if it's running. But it will start if not running.

It's kinda like ensure_modules, but easier to migrate from `dynamic_modules:restart`.

Should be ensure_modules though eventually everywhere